charts/alertmanager-snmp-notifier/values.yaml

```diff
@@ -0,0 +1,185 @@
+restartPolicy: Always
+
+kind: Deployment
+
+image:
+  repository: maxwo/snmp-notifier
+  tag: v1.4.0
```

Comment:
i would rather use appversion var from Chart.yaml.

Example:

https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-conntrack-stats-exporter/values.yaml#L6-L7
https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-conntrack-stats-exporter/templates/daemonset.yaml#L35

Reply:
Done, thanks for the info!